README.md

```diff
@@ -102,6 +117,8 @@ If you see `TestGetManagerWithInvalidUser/Nil_in_user` fail, [try removing](http
 6. run the tests
     ```
     cd testrunner
+    # if on Ubuntu, for instance:
+    apt install php-xml php-curl composer
```

Comment:
I wonder where we should put these PHP requirements for the test suite?
This is a one-time command to get this installed on the local system.
Maybe put a note before step 1 in this list?